core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -297,14 +336,60 @@ private void createSchemaTopic() throws StoreInitializationException {
                + desiredReplicationFactor + ". If this is a production environment, it's "
                + "crucial to add more brokers and increase the replication factor of the topic.");
     }
-    Properties schemaTopicProps = new Properties();
-    schemaTopicProps.put(LogConfig.CleanupPolicyProp(), "compact");
 
+    NewTopic schemaTopicRequest = new NewTopic(topic, 1, (short) schemaTopicReplicationFactor);
+    schemaTopicRequest.configs(Collections.singletonMap(LogConfig.CleanupPolicyProp(), "compact"));
```

Comment:
I only see a CleanupPolicy in streams, but I've updated to use a constant from `TopicConfig`.